app/controllers/application_controller.rb

```diff
@@ -52,10 +52,8 @@ def filtered_records
     # Count Second Opinions here for every record
     @second_opinion_counts = RecordData.where(record: @records, opinion: true).group(:record_id).count
 
-    @records = @records.includes({ingest: :user}, :reviews)
-
-    # Version ID, Version, or Feedback records are the only field we need to render on the home page, so just load that
-    @records = @records.includes(:record_datas).where(record_data: { column_name: ['VersionId', 'Entry_ID/Version', 'Version']}).or(@records.includes(:record_datas).where(record_data: {feedback:true})).references(:record_data)
+    # only include collection records
+    @records = @records.where(recordable_type: 'Collection').distinct
```

Comment:
so do you not need to care about records with `feedback: true` anymore? or is it that since this is grabbing all collections those get picked up also?

Reply:
yeah, I found it odd that is how they classified something to be a "collection" if it had one of those fields, so I asked Jeanne, she is not sure either.   The side effect of that approach was that feedback records didn't show up, hence the ticket.  This is a much cleaner way of doing it, this way all collection records show up regardless of if they have a version or not.   There is some history to this, as I think they used to include BOTH granules and collections in this list, then they decided to filter it this way.   Now it is just granules, maybe by preloading version details made it faster, I'm not sure.   The sql query this generates is pretty efficient and it does lazy loading, so I can't imagine it would be much slower.   Only way to tell I guess would be to grab a snapshot of the current db and try it out... ?

Reply:
Also cmr enforces collection have a version, so we "shouldn't" see any unexpected results.

Reply:
I don't know for sure, but I can't imagine they will have so many collections that are under review at a time that the efficiency of the different sql queries will make a big difference. I was just checking that the right data was getting retrieved.